geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1965,7 +1980,12 @@
    * <U>Default</U>: 6379
    * </p>
    * <U>Allowed values</U>: 0..65535
+   *
+   * <p>
+   * Experimental: Geode API compatible with Redis is a new addition to Geode and the API may change
+   * <p/>
    */
+
   String REDIS_PORT = "compatible-with-redis-port";
```

Comment:
Here we will need the `@Experimental` tag

Reply:
added-  thx